examples/morpheusvm/vm/client.go

```diff
@@ -118,10 +117,6 @@ func (*Parser) AuthCodec() *codec.TypeParser[chain.Auth] {
 	return AuthParser
 }
 
-func (*Parser) StateManager() chain.StateManager {
-	return &storage.StateManager{}
-}
-
```

Comment:
Really nice to be able to remove this from the client where it didn't make much sense

Reply:
Done